README.md

```diff
@@ -113,3 +151,42 @@ For support, please @ David Calder in the [volunteer-app](https://scottishtechar
+
+    b. `versionName` - make this the same as `version` in `/app/package.json`, this should be in quotes (e.g. `"1.0.24"` becomes `"1.0.25"`)
+
+3. Add changelog notes - a quick summary (a line or two will usually do) of what this new version does.  This should be a .txt file in `/app/android/fastlane/metadata/android/en-GB/changelogs` named `X.txt` where `X` is the same number you used for `versionCode` in the previous step, e.g. `/app/android/fastlane/metadata/android/en-GB/changelogs/49.txt`
+
+4. Get your pull request approved as you normally would.  When you're ready to merge your code into the `main` branch and deploy the updated app, double-check your version numbers in the previous steps are still right compared to what's in `main` (somebody else could have merged in code recently and changed the version numbers since you last checked - if you need to, update the version numbers before merging).
+
+    ## Google Play Store (Android)
+
+5. In `/app/src/Config/index.ts` set `STA_BASE_URL` to point to the external URL for [the API endpoint on AWS](#api-deployment-on-aws) -- not to your localhost or its IP address.
+
+6. Go to the `/app/android` directory in a terminal window and run the command `fastlane beta`.  You'll be prompted twice at the beginning for passwords -- both are the password you created in the [Setup to deploy the app section](#setup-to-deploy-the-app) above.
+    > The process can take a while (sometimes 30 minutes or more)!  If it fails, try [the troubleshooting tips here](https://thecodingmachine.github.io/react-native-boilerplate/docs/BetaBuild/#troubleshooting), see [Google Play Store known issues](#google-play-store-known-issues) below or ask for help on the team Slack channel if you can't figure it out.
+
+7. If you have access, check in the [Google Play Console](https://play.google.com/console) that the new version of the app has successfully been added (Volunteer app > Release > Internal testing) -- you should see the new version number next to 'Latest release' under 'Track summary'.
+
+8. Download the updated version of the app to your Android phone ([see download instructions](#download-the-app) near the top of this README).  On the Google Play Store screen, there should be an 'Update' button to download the latest version of the app to your device.
+
+    ### Google Play Store known issues
+
+    - Near the end of the deployment process an AAB file is uploaded to the Google Play Store.  This can take some time (e.g. 20 minutes on slow internet connections).  It should be working, unless you get a `HTTPClient::SendTimeoutError: execution expired` error message in your terminal window.  If the Fastlane process fails for this reason, you may need to run it again.
+
+    ## TestFlight (iOS)
+
+9. In `/app/src/Config/index.ts` set `STA_BASE_URL` to point to **the IP address** for the external URL for the API endpoint on AWS, see [known issues](#known-issues) above -- not to your localhost or its IP address.
```

Comment:
Ah I didn't know you could just use the IP address instead here, I ended up making it so it could connect over HTTP instead for the iOS app, but we need to look at getting a certificate added so we don't need those workarounds

Reply:
Yeah, I'm just going on the notes David Calder added after he set up the API AWS deployment and he was testing on an iPhone.  I did the same as you on Android making it able to connect over HTTP, but we need it to work over HTTPS.  I added a ticket on this the other week that can hopefully get picked up soon: https://sta2020.atlassian.net/browse/SVA-200

Reply:
Nice one, cheers!